### ISSUE TITLE
Build failed

### ISSUE THREAD

**Issue by RayyanNafees** [Bwc9876/Djazztro#3]
it's a freshly installed project from create-djazztro package
I did the `pnpm install` & `pnpm django migrate`

But when i ran `pnpm build`, error occured

![image](https://github.com/Bwc9876/Djazztro/assets/59685062/de679a04-bc10-42a1-af25-fb4610ee6f41)

Djazztro version: 0.0.5

**Comment by Bwc9876** [comment]
I'll fix this in the next version of create-djazztro, but what you can do now is change the astro command in package.json:

```diff
- "astro": "cd frontend && astro"
+ "astro": "astro --root=frontend"
```

**Comment by Bwc9876** [comment]
Fixed in create-djazztro 0.0.14, for anyone who used an older version please do the fix above.
